registry/infrastructure/repositories/base_repository.py

```diff
@@ -1,24 +1,31 @@
-from sqlalchemy import create_engine
+from sqlalchemy import create_engine, text
+from sqlalchemy.exc import OperationalError
 from sqlalchemy.orm import sessionmaker
 from common.logger import get_logger
-from registry.config import NETWORKS, NETWORK_ID
+from registry.config import DB_DETAILS
 
-engine = create_engine(
-    f"{NETWORKS[NETWORK_ID]['db']['DB_DRIVER']}://{NETWORKS[NETWORK_ID]['db']['DB_USER']}:"
-    f"{NETWORKS[NETWORK_ID]['db']['DB_PASSWORD']}"
-    f"@{NETWORKS[NETWORK_ID]['db']['DB_HOST']}:"
-    f"{NETWORKS[NETWORK_ID]['db']['DB_PORT']}/{NETWORKS[NETWORK_ID]['db']['DB_NAME']}", echo=True)
+driver = DB_DETAILS["driver"]
+user = DB_DETAILS["user"]
+password = DB_DETAILS["password"]
+host = DB_DETAILS["host"]
+port = DB_DETAILS["port"]
+db_name = DB_DETAILS["name"]
 
+connection_string = f"{driver}://{user}:{password}@{host}:{port}/{db_name}"
+engine = create_engine(connection_string, echo=True)
```

Comment:
Does echo=True print the connection string? If so please set it to false.

Reply:
echo=True is for enabling logs and it does not print the connection string.